Index: src/Pages/DescriptionPage/DescriptionPage.js
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import React from 'react';\r\nimport css from './DescriptionPage.module.css';\r\nimport {useSelector} from 'react-redux';\r\n\r\nconst DescriptionPage = () => {\r\n    const {oneRestaurant} = useSelector(state => state['placeReducers']);\r\n\r\n\r\n    return (\r\n        <div className={css.desc_page}>\r\n            {oneRestaurant.desc_p_1}\r\n            <img\r\n                src={`https://drive.google.com/uc?export=view&id=${oneRestaurant.desc_p_1_img}`}\r\n                alt={oneRestaurant.name}\r\n            />\r\n        </div>\r\n    );\r\n};\r\n\r\nexport {DescriptionPage};
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/Pages/DescriptionPage/DescriptionPage.js b/src/Pages/DescriptionPage/DescriptionPage.js
--- a/src/Pages/DescriptionPage/DescriptionPage.js	
+++ b/src/Pages/DescriptionPage/DescriptionPage.js	
@@ -8,12 +8,54 @@
 
     return (
         <div className={css.desc_page}>
-            {oneRestaurant.desc_p_1}
-            <img
-                src={`https://drive.google.com/uc?export=view&id=${oneRestaurant.desc_p_1_img}`}
-                alt={oneRestaurant.name}
-            />
+            <div className={css.desc_P}>
+                {oneRestaurant.desc_P}
+            </div>
+            <div className={css.desc_page1}>
+                <div className={css.desc_page_text}>
+                    {oneRestaurant.desc_p_1}
+                </div>
+
+                <div className={css.desc_page_photo}>
+                    <img
+                        src={`https://drive.google.com/uc?export=view&id=${oneRestaurant.desc_p_1_img}`}
+                        alt={oneRestaurant.name}
+                    /></div>
+            </div>
+            <div className={css.desc_page1}>
+
+                <div className={css.desc_page_photo}>
+                    <img
+                        src={`https://drive.google.com/uc?export=view&id=${oneRestaurant.desc_p_2_img}`}
+                        alt={oneRestaurant.name}
+                    /></div>
+                <div className={css.desc_page_text}>
+                    {oneRestaurant.desc_p_2}
+                </div>
+            </div>
+            <div className={css.desc_page1}>
+                <div className={css.desc_page_text}>
+                    {oneRestaurant.desc_p_3}
+                </div>
+                <div className={css.desc_page_photo}>
+                    <img
+                        src={`https://drive.google.com/uc?export=view&id=${oneRestaurant.desc_p_3_img}`}
+                        alt={oneRestaurant.name}
+                    /></div>
+            </div>
+            <div className={css.desc_page1}>
+                <div className={css.desc_page_photo}>
+                    <img
+                        src={`https://drive.google.com/uc?export=view&id=${oneRestaurant.desc_p_4_img}`}
+                        alt={oneRestaurant.name}
+                    /></div>
+                <div className={css.desc_page_text}>
+                    {oneRestaurant.desc_p_4}
+                </div>
+            </div>
+
         </div>
+
     );
 };
 
